client/api/index.js

```diff
@@ -1,3 +1,5 @@
+/* global Stripe */
```

Comment:
[Per Stripe](https://stripe.com/docs/js/including):

> Include the Stripe.js script on each page of your site—it should always be loaded directly from https://js.stripe.com, rather than included in a bundle or hosted yourself.

But then just below on the same page, they also say:

> We also provide an npm package that makes it easier to load and use Stripe.js as a module. For more information, check out the [project on GitHub](https://github.com/stripe/stripe-js).

Then on the docs for that project, it again emphasizes loading it from js.stripe.com:

> Note: To be [PCI compliant](https://stripe.com/docs/security/guide#validating-pci-compliance), you must load Stripe.js directly from https://js.stripe.com. You cannot include it in a bundle or host it yourself. This package wraps the global Stripe function provided by the Stripe.js script as an ES module.

Looking at the code for that module / npm package, it looks like it essentially just provides a wrapper that ultimately still just (lazy?) loads the actual stripe.js source from js.stripe.com.

tl;dr: Seems like it probably doesn't make any tangible difference.

Reply:
> Seems like it probably doesn't make any tangible difference

Yeah, I don't think there is implementation-wise. We're probably fine as-is 🙂